src/main/java/group1/mips_simulator/simulator/programs/Program1.txt

```diff
@@ -0,0 +1,7 @@
+/*
+A program that reads 20 numbers (integers) from the keyboard, prints the numbers to the console printer, requests a
+number from the user, and searches the 20 numbers read in for the number closest to the number entered by the user.
+Print the number entered by the user and the number closest to that number. Your numbers should not be 1…10, but
+distributed over the range of 0 … 65,535. Therefore, as you read a character in, you need to check it is a digit,
+convert it to a number, and assemble the integer.
+*/
```

Comment:
Just so I'm following, these `.txt` files will be binary files? Or assembly to be compiled down into binary?

Reply:
i believe assembly to be compiled down to binary, but honestly jury's out on this
ill ask the prof today

Reply:
think of these more as placeholders for now, we'll replace them with what we actually need when he explains to us what the programs are supposed to be